plug.vim

```diff
@@ -1216,9 +1215,11 @@ class Command(object):
 
     try:
       tfile = tempfile.NamedTemporaryFile(mode='w+b')
+      preexec_fn = not G_IS_WIN and os.setsid or None
       self.proc = subprocess.Popen(self.cmd, cwd=self.cmd_dir, stdout=tfile,
-                                   stderr=subprocess.STDOUT, shell=True,
-                                   preexec_fn=os.setsid)
+                                   stderr=subprocess.STDOUT,
+                                   stdin=subprocess.PIPE, shell=True,
```

Comment:
@mattn Are you sure about that? My understanding is that it should inherit the parents file descriptors which should be set to null, since stdin never gets set. When I took it out, the python installer works fine with my GVim. Seems weird.